### DISCUSSION TITLE
V3.11.2 Meetings with NO District (Group) POSSIBLY NOT Appearing in Meeting Guide

### DISCUSSION THREAD

**Discussion by lloy0076** [code4recovery/12-step-meeting-list#461] [Q&A] [answered]
**Describe the bug**
I think any meeting that has no District 42 association is not appearing in the meeting guide.

**To Reproduce**
1. Install 3.11.2
2. Get a meeting in some state such that it has no "District" entry in the JSON cache
3. I think you do (2) by having a meeting NOT be associated with a group

```
  {
    "id": 2663,
    "name": "Open Discussion",
    "slug": "open-discussion-2",
    "updated": "2021-07-04 10:24:25",
    "location_id": 2664,
    "url": "https:\/\/aa-stlawrenceny.org\/meetings\/open-discussion-2\/",
    "day": 0,
    "time": "10:30",
    "end_time": "11:30",
    "time_formatted": "10:30 am",
    "attendance_option": "in_person",
    "types": [
      "D",
      "O"
    ],
    "location": "South Colton",
    "location_notes": "West off Route 56.",
    "location_url": "https:\/\/aa-stlawrenceny.org\/locations\/south-colton\/",
    "formatted_address": "9 Sugarbush Ln, South Colton, NY 13687, USA",
    "latitude": 44.5089377999999982193912728689610958099365234375,
    "longitude": -74.88738530000000537256710231304168701171875,
    "region_id": 36,
    "region": "Eastern Ontario 83"
  },```

That meeting does not appear in the meeting guide.

```
  {
    "id": 2653,
    "name": "Canton Saturday Night Live",
    "slug": "canton-saturday-night-live",
    "notes": "Because of current guidelines, attendees will be asked to: bring a mask, practise social distance and use hand sanitizer when appropriate.\n\nA full statement can be found at the following URL:\n\nhttps:\/\/aa-stlawrenceny.org\/canton-saturday-night-group-reopening-requirements\/.",
    "updated": "2021-07-01 09:58:17",
    "location_id": 2268,
    "url": "https:\/\/aa-stlawrenceny.org\/meetings\/canton-saturday-night-live\/",
    "day": 6,
    "time": "20:00",
    "end_time": "21:00",
    "time_formatted": "8:00 pm",
    "attendance_option": "in_person",
    "types": [
      "D",
      "O"
    ],
    "location": "Canton",
    "location_notes": "Parking is available in rear of church.",
    "location_url": "https:\/\/aa-stlawrenceny.org\/locations\/uu-canton\/",
    "formatted_address": "3 1\/2 E Main St, Canton, NY 13617, USA",
    "latitude": 44.59626449999999664441929780878126621246337890625,
    "longitude": -75.1678313999999971883880789391696453094482421875,
    "region_id": 36,
    "region": "Eastern Ontario 83",
    "group_id": 2654,
    "group": "Canton Saturday Night Group",
    "district": "District 42",
    "district_id": 37
  },```

This meeting does.

**Expected behavior**
Shouldn't it appear on the meeting guide app?

--> The web-site is live at https://aa-stlawrenceny.org/meetings/?tsml-day=any
--> This meeting DOES NOT appear https://aa-stlawrenceny.org/locations/south-colton/
--> This meeting DOES APPEAR: https://aa-stlawrenceny.org/meetings/fall-island-big-book-study/

It is possible that a meeting with NO group is not being listed in the meeting guide?

I attach the current JSON cache as a zip:

[tsml-cache-5202e6201c.zip](https://github.com/code4recovery/12-step-meeting-list/files/6836543/tsml-cache-5202e6201c.zip)

**Comment by tech2serve** [comment, marked as answer]
Hi @lloy0076,

The answers to your questions can be summarized to say that any meeting that is in the website's JSON feed should show up in Meeting Guide *unless* they have set filters/fences or something isn't correct with the data; however, with 3.11.2, the likelihood of that being the case is less than with previous version because we've improved our validation of data.

So, what someone should do who wants to check this is to actually look at the live JSON feed (not the cache) from your site. One way to do this is to go to Meetings --> Import & Settings, and on the right you'll see an option for "Sharing". Set that to open, and a link will be shown to you to your JSON feed. Click on that, and then search for the meeting in question.

If the meeting is present in the feed, then it is being provided to Meeting Guide and you'll need to reach out to GSO to determine why it isn't showing in Meeting Guide.

If the meeting is not there, then make sure it isn't in DRAFT mode in your database. If it is indeed published, then come back here with the public link and we'll investigate further.

**Reply by lloy0076** [reply]
https://aa-stlawrenceny.org/wp-admin/admin-ajax.php?action=meetings

https://aa-stlawrenceny.org/meetings/open-discussion-2/ | it is in the feed

![image](https://user-images.githubusercontent.com/1174532/126099810-18e80851-7d4f-4d67-98c2-62a96c21cb37.png)

There are other meetings that are not listed and as far as I can tell,

**Reply by joshreisner** [reply]
The listing looks good. Did you reach out to appsupport@aa.org yet? My guess it's either a conflict with Area 48 or it's outside the boundary they set. You could ask them to see a screenshot of the boundary…

**Reply by lloy0076** [reply]
@joshreisner I sent them a message at 8:39AM but am yet to hear a response; they used to have an auto responder saying something like COIVD19 delayed the response time but that didn't trigger so we'll see what they say.

Message I sent was:

> 
> Our district registrar has noticed that the South Colton meeting is not on the Meeting Guide.
> 
>     https://github.com/code4recovery/12-step-meeting-list/discussions/461#discussioncomment-1019803
> 
> --> The web-site is live at https://aa-stlawrenceny.org/meetings/?tsml-day=any
> --> This meeting DOES NOT appear https://aa-stlawrenceny.org/locations/south-colton/
> --> This meeting DOES APPEAR: https://aa-stlawrenceny.org/meetings/fall-island-big-book-study/
> 
> They have said:
> 
> Hi @lloy0076,
> 
>     The answers to your questions can be summarized to say that any meeting that is in the website's JSON feed should show up in Meeting Guide unless they have set filters/fences or something isn't correct with the data; however, with 3.11.2, the likelihood of that being the case is less than with previous version because we've improved our validation of data.
> 
>     So, what someone should do who wants to check this is to actually look at the live JSON feed (not the cache) from your site. One way to do this is to go to Meetings --> Import & Settings, and on the right you'll see an option for "Sharing". Set that to open, and a link will be shown to you to your JSON feed. Click on that, and then search for the meeting in question.
> 
>     If the meeting is present in the feed, then it is being provided to Meeting Guide and you'll need to reach out to GSO to determine why it isn't showing in Meeting Guide.
> 
>     If the meeting is not there, then make sure it isn't in DRAFT mode in your database. If it is indeed published, then come back here with the public link and we'll investigate further.
> 
> And verified, too that it is in our feed.
> 
> 
> DSL

**Reply by lloy0076** [reply]
They said:

```
Hi David,

Thank you for contacting App Support. It seems that South Colton was not included in the automated geo-boundary locations for your office. We were able to extend the boundary to now cover South Colton, so the meeting should correctly display on the app. 

Please let us know if you have any questions.


Best,
```

I'm waiting for it to turn up on the meeting application.
